src/http/Message.js

```diff
@@ -125,6 +126,28 @@ export default class Message {
         this.header.finalize(); // after syncContentLength() adds headers
     }
 
+    // copy the 'ID' field from other message
```

Comment:
This ID copying functionality only makes sense for responses, right? It would not even work for request-request because the header names will clash. Should we move this to Response.js and adjust to be more specific to (known) message kinds?